modules/special-rules/pages/flak.adoc

```diff
@@ -1,2 +1,62 @@
+Others simply fill the air with so many munitions that they can hardly fail to get some hits.
+
+---
+
+{flak} units can attack flyers in one of two different ways, depending on what orders you give to their detachment.
+
+[IMPORTANT]
+.Flak and drop pods
+====
+{flak} units may fire on drop pods as if they were flyers, against the Armour value of the transported unit.
+====
+
+== Flak orders
+For optimum effectiveness, {flak} units and their detachment need to be at a suitable state of readiness.
+We represent this state with special orders called Flak.
+
+. If you wish to place any detachments on Flak orders, you must do this in the Start phase.
++
+[TIP]
+====
+You can place a detachment on both Flak orders and Preparatory Bombardment orders at the same time in the Start phase, if you wish, declaring both at the same time.
+
+In this case, you will be able to make a Preparatory Bombardment with {artillery} units in the detachment, and make Flak attacks with {flak} units.
+====
+. Place the {special-orders} marker to indicate that the detachment is on Flak orders.
+. You can use these units to shoot at flyers in the movement phase.
++[TODO: ...and at drop pods? The errata mentions that they can Snap Fire at 'flyers and drop pods' in the 'other orders' bit of text, seemingly as an afterthought -- seems to imply that they should be able to target drop pods while on Flak orders too.]+
++[IJW: It's not spelled out in the original Flak rules, because it's in the drop pod rules. However, see the IMPORTANT box I've added at the top, which spells out that Flak units treat drop pods as flyers.]+
+ * You may not move the units in the Movement phase, nor use them to shoot in the Shooting phase.
+ * Treat the other units in the detachment as though they are on Overwatch orders -- so you can move them up to 5cm in the Movement phase and re-roll misses when you shoot with them in the Shooting phase.
+. {flak} units can shoot at enemy flyers if any part of their flight path is within the {flak} unit's full Range:
+ * Roll one D6 for each point of the {flak} unit's Firepower.
+ Each result that is equal to or higher than the flyer's Armour value scores a hit on the flyer.
+ You can split the firepower of the {flak} unit between several flyers if you wish.
+ * Once you have rolled a D6 for each point of the unit's Firepower, the unit cannot shoot for the rest of the turn.
+ * Super-heavy weapons can be applied to the unit's shooting, for example an Eldar Fire Prism's {macro-weapon} means that it's one shot will hit on a 4+ regardless of the flyer's Armour value. +
+ +[TODO: Should we unify this with the usual hit allocation rules? Seems like we should clarify special abilities too, eg {macro-weapon} which is mentioned in the errata for the alternative Snap Fire attack but not in the main part of the Flak rules.]+
+ +[IJW: As far as I can see, the Flak unit gets to choose targets freely, and yes MW works when on Flak orders.]+
```

Comment:
```suggestion
```
No longer needed. I figured that MW must work on Flak orders, just wasn't sure whether to explicitly mention it.
I think you're right that the player gets to choose freely among valid flyer targets though.